src/pages/help-centre/components/_answer-card.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactNode, useContext } from 'react'
+import styled from 'styled-components'
+import { TAnswer } from '../data/_data-types'
+import ImageCard from './_image-card'
+import TranslationComponents from './_translation-components'
+import List from './_list'
+import device from 'themes/device'
+import { Header } from 'components/elements'
+import { Localize } from 'components/localization'
+import { TString } from 'types/generics'
+import { DerivStore } from 'store'
+
+type AnswerCardType = {
+    label: string
+    question: TString
+    answer?: TAnswer
+    renderProp?: () => ReactNode
+}
+
+export const Wrapper = styled.div`
+    max-width: 71.2rem;
+    margin-left: 12.6rem;
+    margin-top: 1.6rem;
+    height: 100%;
+
+    @media ${device.laptopL} {
+        margin-left: 2rem;
+    }
+
+    @media ${device.tabletL} {
+        margin-left: 0;
+    }
+`
+
+const AnswerCard = ({ question, answer, renderProp }: AnswerCardType) => {
+    const { is_eu_country } = useContext(DerivStore)
+
+    return (
+        <Wrapper>
+            <Header size="2.4rem" mb="2.4rem">
+                <Localize translate_text={question} />
+            </Header>
+            {renderProp && renderProp()}
+            {answer &&
+                answer.map(
+                    ({
+                        translation_text,
+                        eu_translation_text,
+                        translation_components,
+                        has_margin_top,
+                        margin_top,
+                        list,
+                        img,
+                    }) => (
+                        <>
+                            <Header
+                                key={translation_text}
+                                size="16px"
+                                weight="normal"
+                                mt={has_margin_top ? '1.7rem' : margin_top}
+                            >
+                                {translation_text && (
+                                    <Localize
+                                        translate_text={
+                                            is_eu_country && eu_translation_text
+                                                ? eu_translation_text
```

Comment:
same for `eu_translation_text`

Reply:
In some of our arrays we have both **eu_translation_text** and **translation_text** because we are showing different texts in different region!
The reason for this condition is that, I'm checking that if the country is **eu** and we have **eu_translation_text** just use this text else use **translation_text**